openwisp_firmware_upgrader/static/firmware-upgrader/css/upgrade-progress.css

```diff
@@ -41,14 +41,155 @@
   background-color: #efb80b;
 }
 
+.upgrade-progress-fill.cancelled {
+  background-color: #efb80b;
+}
+
 .upgrade-progress-fill.in-progress {
   background-color: #70bf2b;
 }
 
 .upgrade-progress-text {
   color: #000;
-  font-weight: normal;
+  font-weight: bold;
   font-size: 12px;
   display: inline-block;
   vertical-align: middle;
 }
+
+.upgrade-cancel-btn {
+  background-color: #dc3545;
+  color: white;
+  border: none;
+  padding: 8px 16px;
+  border-radius: 4px;
+  font-size: 11px;
+  cursor: pointer;
+  margin-left: 10px;
+}
```

Comment:
Let's maintain `cancel btn` red color consistency (this one looks different than other pages)

<img width="1345" height="703" alt="Screenshot from 2025-07-30 16-40-54" src="https://github.com/user-attachments/assets/2c15f85a-30b0-4b44-b139-5c4b6e0238c4" />

<img width="480" height="270" alt="Screenshot from 2025-07-30 16-43-21" src="https://github.com/user-attachments/assets/c5394966-f176-4e29-bb84-b4182832ad95" />